chromium_src/chrome/common/importer/profile_import_process_param_traits_macros.h

```diff
@@ -1,3 +1,8 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
This file fails on linter with this error:

```
chromium_src/chrome/common/importer/profile_import_process_param_traits_macros.h:0:  No #ifndef header guard found, suggested CPP variable is: BRAVE_CHROMIUM_SRC_CHROME_COMMON_IMPORTER_PROFILE_IMPORT_PROCESS_PARAM_TRAITS_MACROS_H_  [build/header_guard] [5]
```

I tried fixing it with adding guards, but then it fails. I talked with @bsclifton about it and we decided to leave it for now as it's not part of this PR and would probably need some patching.

cc @bridiver

Reply:
remove as it's not needed